pkgs/applications/science/misc/root/default.nix

```diff
@@ -57,11 +57,13 @@ in
 
 stdenv.mkDerivation rec {
   pname = "root";
-  version = "6.26.06";
+  version = "6.26.08";
 
-  src = fetchurl {
-    url = "https://root.cern.ch/download/root_v${version}.source.tar.gz";
-    hash = "sha256-sfc8l2pYClxWyMigFSWCod/FYLTdgOG3VFI3tl5sics=";
+  src = fetchFromGitHub {
```

Comment:
Ant specific reason to use `fetchFromGitHub`(`fetchzip`)? Nice thing about `fetchurl` is that it mirrors to `tarballs.nixos.org`.

Reply:
This release is only tagged on GitHub. Meanwhile, Fedora and OpenSUSE have all bumped to this version.